### DISCUSSION TITLE
Tutorial for relation extraction using spacy and LLM

### DISCUSSION THREAD

**Discussion by sameerdevstack** [explosion/spacy-llm#266] [Q&A] [answered]
Hi,
can someone provide a detailed tutorial on how to relation extraction model using LLM and spacy for a beginner ?
even if you just mention the steps from the start (instead of full explanation ) it will be fine .

Please help me to understand the method and procedure
Thanks and regards !

**Comment by rmitsch** [comment, marked as answer]
Hi @sameerdevstack, check out our [relation extraction example with `spacy-llm`](https://github.com/explosion/spacy-llm/tree/main/usage_examples/rel_openai). This should be enough to get you started. If you have more specific question, feel free to open another thread!

**Reply by sameerdevstack** [reply]
Thanks for the reply.
I will follow this.
